### PR TITLE
add devtools + combine tests

### PR DESCRIPTION
**Pull request by rosostolato**
Created tests for devtools + combine

### CODE REVIEW

**Comment by dai-shi** [comment]
Note: This is continuing from #230.

File: tests/middleware.test.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          send(action: string, payload: any) {
+            expect(action).toBe('action')
+            expect(payload.count).toBe(1)
+            expect(typeof payload.inc).toBe('function')
+          },
+        }
+      },
+    }
+
+    const useStore = create<any>(
```
Comment (dai-shi) [inline tests/middleware.test.tsx:28]: I guess one of our goals is to type correctly (cf. #216)
Can we remove `<any>` for `create()` or type it properly?

**Review by dai-shi — COMMENTED** [review]
Thanks for your work.

File: tests/middleware.test.tsx (hunk 1 above)
Comment (rosostolato) [reply tests/middleware.test.tsx:28]: sure

File: tests/middleware.test.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            )
+          },
+          send(action: string) {
+            expect(action).toBe(`${storeName} > ${actionName}`)
+          },
+        }
+      },
+    }
+
+    const useStore = create<any>(
```
[the remark this answers is not in the document]
Comment (dai-shi) [reply tests/middleware.test.tsx:151]: We were hoping #222 to handle this, but seems it does not. Hm. 🤔

[the remark this answers is not in the document]
Comment (dai-shi) [reply tests/middleware.test.tsx:151]: OK, I think I should revert #222, as it was a wrong fix. We should probably start over. I don't even know if it's technically possible.

**Comment by dai-shi** [comment]
Closing this as stale. I hope someone could continue the challenge.

**Pull request closed without merging**